app/scripts/lib/rpc-method-middleware/handlers/wallet-createSession/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum KnownSessionProperties {
+  SolanaAccountChangedNotifications = 'solana_accountChanged_notifications',
+}
+
+export function isKnownSessionPropertyValue(
+  value: string,
+): value is KnownSessionProperties {
+  return Object.values(KnownSessionProperties).includes(
+    value as KnownSessionProperties,
+  );
+}
```

Comment:
should this live in core?

Reply:
`KnownSessionProperties` does: https://github.com/MetaMask/core/blob/383933c11ce7055f8d5d76ec3008778470710975/packages/chain-agnostic-permission/src/scope/constants.ts#L96

I had just forgotten to export it previously

Reply:
Working on this now

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/29887/commits/47bb0f7e41d020d8b7aacc2118478b8041d5bf9b